src/components/select/SelectReps/index.tsx

```diff
@@ -58,13 +65,6 @@ function useSelectRepsAPI(props: SelectRepsProps) {
   };
 }
 
-// TODO - Consider updating this where the current rep choice always has a
```

Comment:
I thought about it and agreed with one of the past "me"s, this would be annoying because of the bumping around so removing it as a TODO